dapps/shop/src/pages/admin/settings/Console.js

```diff
@@ -132,6 +276,32 @@ const AdminConsole = () => {
           Submit
         </button>
       </form>
+
+      <label className="mt-4 font-weight-bold">Emit ListingUpdated event</label>
+      <form
+        className="d-flex"
+        onSubmit={async e => {
+          e.preventDefault()
+          if (!shopIpfsHash) {
+            return
+          }
+          console.log('Calling ListingUpdated...')
+          updateListing(config, shopIpfsHash)
+            .then(() => console.log('Listing updated successfully'))
```

Comment:
Should this maybe have more user feedback?

Reply:
Yes, all the functionalities exposed in the console are pretty bare bone and don't provide any UI feedback. Probably ok for the demo but in the future we if the business team starts using those, we should add some feedback.

For my information, do we have some toast library for showing a success/error message in the UI that we already use in the shop admin? I did not see any...